spatialdata/_io/format.py

```diff
@@ -52,19 +52,100 @@ def validate_shapes_parameters(
             if len(transforms["scale"]):
                 raise ValueError("`coordinate_transformations` must contain a `scale` of length 0.")
 
-    def validate_tables(
+    def validate_table(
         self,
-        tables: AnnData,
+        table: AnnData,
         region_key: Optional[str] = None,
         instance_key: Optional[str] = None,
     ) -> None:
-        if not isinstance(tables, AnnData):
+        if not isinstance(table, AnnData):
             raise ValueError("`tables` must be an `anndata.AnnData`.")
         if region_key is not None:
-            if not is_categorical_dtype(tables.obs[region_key]):
+            if not is_categorical_dtype(table.obs[region_key]):
                 raise ValueError(
-                    f"`tables.obs[region_key]` must be of type `categorical`, not `{type(tables.obs[region_key])}`."
+                    f"`tables.obs[region_key]` must be of type `categorical`, not `{type(table.obs[region_key])}`."
                 )
         if instance_key is not None:
-            if tables.obs[instance_key].isnull().values.any():
+            if table.obs[instance_key].isnull().values.any():
                 raise ValueError("`tables.obs[instance_key]` must not contain null values, but it does.")
+
+    def generate_coordinate_transformations(self, shapes: List[Tuple[Any]]) -> Optional[List[List[Dict[str, Any]]]]:
+
+        data_shape = shapes[0]
+        coordinate_transformations: List[List[Dict[str, Any]]] = []
+        # calculate minimal 'scale' transform based on pyramid dims
+        for shape in shapes:
+            assert len(shape) == len(data_shape)
+            scale = [full / level for full, level in zip(data_shape, shape)]
+            from spatialdata._core.transformations import Scale
+
+            coordinate_transformations.append([Scale(scale=scale).to_dict()])
+        return coordinate_transformations
+
+    def validate_coordinate_transformations(
```

Comment:
Yes, I think we can remove it here. I think this was a leftover from taking the formatter from ngff

Reply:
I think is not needed to add a validation for coordinate transformations and coordinate systems because we are not reusing existing types for storing the information (like `AnnData`, `DataArray` etc) but instead we use a constructor to initialize the objects (even when we call `from_dict()`).